RoslynPluginGenerator/AnalyzerPluginGenerator.cs

```diff
@@ -72,11 +72,17 @@ public bool Generate(NuGetReference analyzeRef, string language, string sqaleFil
             }
             SupportedLanguages.ThrowIfNotSupported(language);
 
-            string nuGetDirectory = Utilities.CreateTempDirectory(".nuget");
-            IPackage package = this.packageHandler.FetchPackage(analyzeRef.PackageId, analyzeRef.Version, nuGetDirectory);
+            IPackage package = this.packageHandler.FetchPackage(analyzeRef.PackageId, analyzeRef.Version);
 
             if (package != null)
             {
+                // Build machines will need to install this package, it is not feasible to create plugins for packages requiring license acceptance
+                if (PackageRequiresLicenseAcceptance(package)) 
+                {
+                    this.logger.LogError(UIResources.APG_NGPackageRequiresLicenseAcceptance);
```

Comment:
Would it help the user to include the package id and version in the error message?